packages/assets-controllers/src/token-prices-service/codefi-v2.test.ts

```diff
@@ -74,13 +74,28 @@ describe('CodefiTokenPricesServiceV2', () => {
           },
         });
 
-      await expect(
-        new CodefiTokenPricesServiceV2().fetchTokenPrices({
-          chainId: '0x1',
-          tokenAddresses: ['0xAAA', '0xBBB', '0xCCC'],
+      const result = await new CodefiTokenPricesServiceV2().fetchTokenPrices({
+        chainId: '0x1',
+        tokenAddresses: ['0xAAA', '0xBBB', '0xCCC'],
+        currency: 'ETH',
+      });
+      expect(result).toStrictEqual({
+        '0xAAA': {
+          tokenAddress: '0xAAA',
+          value: undefined,
```

Comment:
Hmm, I'm a bit uneasy using `undefined` as the price of a token here. It seems that if we don't have a price, then this object shouldn't even show up in this object at all. Would that create any problems?

If we did this, then I think we will want to change the return type of `fetchTokenPrices` to `Partial<TokenPricesByTokenAddress<Hex, SupportedCurrency>>` instead of `TokenPricesByTokenAddress<Hex, SupportedCurrency>` (and we will want to make a similar change to `AbstractTokenPricesService`). We can also remove the type assertion on line 368 of `codefi-v2.ts`.

Reply:
Hey @mcmire , wasn't sure if we would rather have the whole object to be removed from the response or just have an empty object instead ex: 
`'0xAAA': { } `

I have made updates on the code; to have the contract address but the object will be empty if the price is undefined.
Sorry for the back and forth; please let me know if anything 🙏